assets/js/guides.js

```diff
@@ -120,4 +120,19 @@ $(document).ready(function () {
     ga('send', 'event', location.pathname, 'subscribe-cta');
   })
 
+  //Remove link from parent navigation item and add classes when clicked
+  $.each($("#toc ul > li"), function (index, obj) {
+    if ($(obj).has("ul").length) {
```

Comment:
Minor nit: `.length > 0` is more readable and relies less on magical truthiness in JS. Also, store the output of `$(obj).has("ul")` in a variable so it can be reused below instead of copy/pasted.